src/lib/components/mods/ModInfo.svelte

```diff
@@ -4,29 +4,44 @@
   import { prettyDate, prettyNumber } from '$lib/utils/formatting';
 
   import TagList from '$lib/components/utils/TagList.svelte';
-  import CompatibilityButton from './compatibility/CompatibilityButton.svelte';
 
   export let mod!: Pick<
     Mod,
     'source_url' | 'created_at' | 'views' | 'downloads' | 'mod_reference' | 'compatibility' | 'tags'
   >;
+
+  function urlWrap(url: string) {
+    if (url.length > 40) {
+      return url.replace(/\/(?!.*\/)/g, '/\n');
+    }
+    return url;
+  }
```

Comment:
It can be done, but I think we might have been going for a break at the `/`
![image](https://github.com/satisfactorymodding/smr-frontend/assets/47161774/84462deb-f207-4811-8b46-4bd83f0186c9)

Reply:
Fixed: https://github.com/satisfactorymodding/smr-frontend/pull/81/commits/1a3ac79c47ddcf0994be70c73b08f08191de8f1b